packages/release-please/src/release-please.ts

```diff
@@ -383,13 +385,27 @@ async function runBranchConfiguration(
       plugins
     );
     try {
-      const numReleases = await Runner.createReleases(manifest);
-      logger.info(`Created ${numReleases} releases`);
-      if (numReleases > 0) {
+      const releases = await Runner.createReleases(manifest);
+      logger.info(`Created ${releases.length} releases`);
+      if (releases.length > 0) {
         // we created a release, reload config which may include the latest
         // version
         manifest = null;
       }
+      if (releases.length > 0 && branchConfiguration.tagPullRequestNumber) {
+        // All releases from one Git push event share the same pull request
+        // number and commit SHA. We just use the first one.
```

Comment:
This may not be true.

Release-please looks for all pending releases and will try and tag them all (not necessarily only the last one). We probably want to find all the unique release PR/commit combinations and tag each of those

Reply:
Thank you.

Reply:
Fixed. Added a test case 'should tag each SHA for pull requests number if configured'.